common/common.scss

```diff
@@ -1,3 +1,22 @@
+@media (min-width: 501px) {
```

Comment:
Ideally I'd re-use the exact same breakpoints from Tailwind, but this is not critical for now (as I shared, the plan is to have a single CSS framework for both our sites)